Makefile

```diff
@@ -35,9 +35,9 @@ endif
 
 # Build the project images
 .DELETE_ON_ERROR:
-all: sensor-linux sensor-controller-linux gateway-controller-linux gateway-client-linux webhook-linux calendar-linux resource-linux artifact-linux file-linux nats-linux kafka-linux amqp-linux mqtt-linux storage-grid-linux github-linux hdfs-linux gitlab-linux sns-linux sqs-linux pubsub-linux trello-linux
+all: sensor-linux sensor-controller-linux gateway-controller-linux gateway-client-linux webhook-linux calendar-linux resource-linux artifact-linux file-linux nats-linux kafka-linux amqp-linux mqtt-linux storage-grid-linux github-linux hdfs-linux gitlab-linux sns-linux sqs-linux pubsub-linux
```

Comment:
Is removing of the trello gateway intended for this PR?

Reply:
Yeah. The library for trello doesn't have solid support for webhooks. e.g. it lets you create webhook but doesn't let you delete it which is a problem when you want to stop an event source. Also, it doesn't take `id` at the time of creation which could have been used later to check if the hook is already created, especially in the case where gateway pod has restarted. 

It would be in vain to refactor trello gateway for RouteConfig interface. So I got rid of it.

If the trello library matures and support to delete hooks, we can add it later.